client/src/modules/dashboard/Chapters/pages/ChaptersPage.tsx

```diff
@@ -50,28 +90,51 @@ export const ChaptersPage: NextPageWithLayout = () => {
   const isLoading = loading || loadingUser || !data;
   if (isLoading || error) return <DashboardLoading error={error} />;
 
+  const filteredChapters = data.dashboardChapters.filter(({ name }) =>
+    name.toLowerCase().includes(search.trim()),
```

Comment:
```suggestion
    name.toLowerCase().includes(search.toLowerCase().trim()),
```
I forgot to add this earlier

Reply:
I should have noticed as well 🤣